### ISSUE TITLE
where can i found checkpoints

### ISSUE THREAD

**Issue by 289282737** [Alik033/Lit-Net#1]
HELLO！Alik033! I have star.
I train the moudle,but the SSIM so low,and MSE so high.
This is my question -------loading model for generator  None
                                          No checkpoints found for netG and netD! retraining

Please answer me,Thank you! I can understand English and Chinese!
C:\ProgramData\anaconda3\envs\test\python.exe D:\Lit-Net-main\Enhancment\train.py 
C:\ProgramData\anaconda3\envs\test\lib\site-packages\torchvision\models\_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
C:\ProgramData\anaconda3\envs\test\lib\site-packages\torchvision\models\_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=VGG16_Weights.IMAGENET1K_V1`. You can also use `weights=VGG16_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
loading model for generator  None
No checkpoints found for netG and netD! retraining
 Epoch : 1 | (17/2287) | mse: 0.8905353546142578 | vgg: 0.159453883767128 | ssim: 0.13122130930423737

**Comment by Alik033** [comment]
Thanks for showing interest in Lit-Net.
I have uploaded the checkpoints for the UIEB and LSUI datasets. I will upload the checkpoints for the remaining datasets shortly. 

Regarding the SSIM and MSE, during training, you see the SSIM and MSE loss value. It is not the SSIM and MSE on the test set. First, you train for a few epochs, test it with a test set, and check the SSIM and MSE.

**Comment by 289282737** [comment]
I have understood. Thank you very much for your reply and assistance!  
&nbsp;


罗凯伦
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Alik ***@***.***&gt;; 
发送时间: 2024年11月19日(星期二) 下午3:34
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Alik033/Lit-Net] where can i found checkpoints (Issue #1)














 
Thanks for showing interest in Lit-Net.
 I have uploaded the checkpoints for the UIEB and LSUI datasets. I will upload the checkpoints for the remaining datasets shortly.
 
Regarding the SSIM and MSE, during training, you see the SSIM and MSE loss value. It is not the SSIM and MSE on the test set. First, you train for a few epochs, test it with a test set, and check the SSIM and MSE.
